wremnants/combine_helpers.py

```diff
@@ -6,6 +6,7 @@
 logger = logging.child_logger(__name__)
 
 def add_scale_uncertainty(card_tool, scale_type, samples, to_fakes, name_append="", resum=None, use_hel_hist=False, rebin_pt=None):
```

Comment:
The code checks 'resum != "none" ', shouldn't it be ' resum != None ' instead ? The default argument is actually None, and "none" string is different from None

Reply:
Yes this is incredibly confusing and error prone.  The default is actually one of the other options, so to allow setting it to none on the command line it has to be a string...